cmd/scheduler/main.go

```diff
@@ -168,6 +168,38 @@ func main() {
 		}
 	}
 
+	// Schedule cleanup tasks
+	retentionPeriod := 5 * 30 * 24 * time.Hour // 30 days
```

Comment:
Not necessary for this round, however for the future ideally this would be configurable.